android/src/main/java/tv/vizbee/rnsender/VizbeeNativeManager.java

```diff
@@ -431,6 +465,13 @@ private WritableMap getSessionConnectedDeviceMap() {
         return map;
     }
 
+    private void notifyGetSignInInfo() {
+
+        Log.v(LOG_TAG, "Sending signin info trigger ...");
+        WritableMap signInInfoMap = Arguments.createMap();
+        this.sendEvent("VZB_INVOKE_GET_SIGNIN_INFO", signInInfoMap);
```

Comment:
Let's use contacts instead of magic numbers or string

Reply:
All magic strings are moved to VizbeeConstants file